ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-conf-aws-sharednetwork
```

Comment:
We do this in a few places already, but I [hear][1] there are some ci-tools issues with using the same name for both a `ref` and a `chain`.  Might be worth using different names for them here; or maybe that's no longer an issue.

[1]: https://github.com/openshift/release/pull/7931#issuecomment-604144637

Reply:
https://github.com/openshift/ci-tools/pull/818